content/en/develop/content/migration-for-content-team.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: MakerDAOEXTREEMME
+---
+# Migration from .md to .mdxfiles
+
+## Setting up Local development
+
+By the time this week completes (in time for America day) some will be developing locally. Which is no small feat and is kind of a badass skill. [Here is Gatsby's guide to doing basically everything in this doc](https://www.gatsbyjs.org/tutorial/) it's elegant, thorough, and helfpul AF. Plus it's like a half hour, tops, of following directions and installing things.
+
+### The terminal 
+
+***:face_with_raised_eyebrow: i see you're reading on...***
+
+If one hasn't done anything before in the terminal, recognize that it's the pinnacle of laziness & elegance. Developers literally have set this thing up to run on as few characters as possible. Our devs, specifically, have set community contributors to use even fewer commands. 
+
+If you're lazy (congratulations :confetti_ball:) , then here the short-short guide to Installs 
+- [VS code](https://code.visualstudio.com/download) 
+    - IDE's with the best text highlighting experience, and development ease/packages/extensions/themes, borders on an obsession. so, download packages/extensions at one's own discretion. Generally it's best to have:
+        - A linter 
+        - A prettier extension 
```

Comment:
They won't need a prettier extension, but it's a nice to have. 

We have prettier built into the project through the ```yarn format``` command. The code is also formatted in the poolparty command so everyone always gets pretty formatted code when they start :)